Dockerfile

```diff
@@ -1,29 +1,34 @@
-# Build container
+# Stage 1: Build container
+# Use a specific Golang version and Alpine Linux as the base image
 ARG GOVERSION=1.20.12
 ARG ALPINEVERSION=3.19
 
 FROM --platform=${BUILDPLATFORM} \
     golang:$GOVERSION-alpine${ALPINEVERSION} AS build
 
+# Set the working directory for the build
 WORKDIR /src
-RUN apk --no-cache add git build-base
 
-ENV GO111MODULE=on \
-    CGO_ENABLED=0
```

Comment:
The placement of these was intentional to improve readability of the Dockerfile.